spec/integration/data_stream/manifest.spec.yml

```diff
@@ -490,6 +490,16 @@ spec:
         dynamic_namespace:
           description: When set to true, agents running this integration are granted data stream privileges for all namespaces of its type
           type: boolean
+    owner:
+      type: object
+      additionalProperties: false
+      properties:
+        github:
```

Comment:
Would we want a `type` also here? This would be useful for example if we expect a data stream to be maintained by a partner, while the package is owned by elastic (or the other way around).